lib/active_model/serializer.rb

```diff
@@ -11,6 +12,7 @@ class Serializer
     autoload :Associations
     include Configuration
     include Associations
+    include KeyFormat
```

Comment:
I like the organization that use external classes bring to the code, but taking into account the complexity of `KeyFormat` Class how do you feel about bring this logic to the `Serializer` itself in other to keep it concise? 
I'd like to heard your thoughts about it :smile: